etc/vscode/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
```

Comment:
Is this required for the extension? I prefer to keep a single changelog for the entire project; there is one in the repository root already.